API.md

```diff
@@ -4581,6 +4581,12 @@ Request information:
 
 Note that the `request.info` object is not meant to be modified.
 
+#### <a name="request.isInjected" /> `request.isInjected`
+
+Access: read only.
+
+`true` if the request was created via [`server.inject`](#server.inject()), and `false` otherwise.
```

Comment:
Sorry, I should have noticed this before. Based on the other uses of `server.inject()` in this file, I think this should be:

```suggestion
`true` if the request was created via [`server.inject()`](#server.inject()), and `false` otherwise.
```